### PR TITLE
Switch off reading of BadChannel CCDB information in StatusMap Creator

### PR DESCRIPTION
**Pull request by lmassacr**
Switch off the reading of BadChannel CCDB information in StatusMap Creator. 
A protection will need to be developped in the code in case an Invalid Solar Id is read from the BadChannel CCDB information, before one can enable back this option in the MCH reconstruction.

### CODE REVIEW

**Comment by chiarazampolli** [comment]
Hello @lmassacr , @shahor02 , @TimoWilken , 

The CI is not starting, I think the MCH should approve first, right?

Cheers,

Chiara

**Comment by TimoWilken** [comment]
Hi @chiarazampolli, yes, someone in the team @AliceO2Group/muon-experts needs to approve this PR for the checks to start.

**Review by shahor02 — APPROVED** [review]
conditionally approving to start CI

**Comment by lmassacr** [comment]
> Hello @lmassacr , @shahor02 , @TimoWilken ,
> 
> The CI is not starting, I think the MCH should approve first, right?
> 
> Cheers,
> 
> Chiara

Hi Chiara @chiarazampolli, 

Indeed, I need to ask to be added in the muon-expert list as replacement for Laurent. I don't have the approval rights yet. 
I think Philippe or Andrea should have. I will ping them. 
In parallel I am still looking into a solution with a patch of the CCDBs (we have a CCDB entry per calibration run in data, so this might explain why with the same code some period/runs are crashing and others not). We need to scan all the CCDB entries to identify the falty ones, also cross checking with the reported crashed jobs. 

Cheers,
Laure

**Review by shahor02 — APPROVED** [review]

**Review by shahor02 — APPROVED** [review]

**Comment by shahor02** [comment]
Before I was able to force-approve, but does it not work anymore, even after adding myself as a codeowner. Indeed need to wait for somebody from the @AliceO2Group/muon-experts

**Comment by TimoWilken** [comment]
No you're right @shahor02, your approval seems to have worked -- some of the checks have started. Your are a repo admin, so you can always approve.

**Comment by pillot** [comment]
> Hi @chiarazampolli, yes, someone in the team @AliceO2Group/muon-experts needs to approve this PR for the checks to start.

Hi @TimoWilken , just for my understanding, is it a new feature ? So far we were waiting for the checks to pass before approving the PR.

**Review by pillot — APPROVED** [review]
Just in case my approval is still a blocker, but I don't think so.

**Comment by chiarazampolli** [comment]
Hello @shahor02 , 
Can this be merged?
Chiara

**Comment by TimoWilken** [comment]
Hi @pillot, while this is not a new feature, this only applies to new contributors' first PRs. This is for security reasons, since the checks can run arbitrary user-supplied code.

Now that this is merged, all of @lmassacr's future PRs will not need approval for the checks to start!

**Comment by pillot** [comment]
> Hi @pillot, while this is not a new feature, this only applies to new contributors' first PRs. This is for security reasons, since the checks can run arbitrary user-supplied code.
> 
> Now that this is merged, all of @lmassacr's future PRs will not need approval for the checks to start!

Ok, good to know, thanks for the explanations! :)

**Pull request merged**
